scripts/lib/build-utils.mjs

```diff
@@ -175,7 +175,7 @@ export default {
         ],
       },
       plugins: [
-        new webpack.EnvironmentPlugin({ NODE_ENV: mode }),
+        new webpack.EnvironmentPlugin({ NODE_ENV: process.env.NODE_ENV }),
```

Comment:
Is `process.env.NODE_ENV` set to `"production"` in the electron runtime environment? Because in the old code `mode` is set to `"production"` by default if the test/dev checks fail.